config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -66,11 +66,6 @@ spec:
                           type: string
                       type: object
                   type: object
-              oneOf:
-              - required:
-                - value
-              - required:
-                - valueFrom
```

Comment:
This PR reverted a change previously made in #684

Reply:
I see, this is going to be replaced each time CRDs are regenereated given that kubebuilder doesn't support it.

Reply:
@detiber do you know any other ways to achieve this apart from a hacky kustomize / bash script?

Reply:
We probably need to create and track an issue against sigs.k8s.io/controller-runtime to get a proper fix in place.